Fix various css issues

alexdryden opened the pull request:
This closes #46, closes #40, closes #10 by:

- using the compiled bootstrap.css instead of the CDN
- adding a clear_fix solution after images in media blocks
- give a more specific rule for the display of .media class divs

alexdryden committed:
add non-minified bootstrap css and remove cdn link

alexdryden committed:
fix styling issues

add a clear class to clear after floating images and add a more
specific rule for displaying media to avoid collision with bootstrap
.media

alexdryden committed:
add a div to clear float after the image

alexdryden committed:
correct type in clearfix class

alexdryden committed:
change to more recent version of bootstrap

Bosterm commented:
Everything works for me except #40. For some reason images are still having the same issue. See: [this page](https://pressbookstest.web.illinois.edu/omeka-s/s/test-site/page/test-2)

alexdryden committed:
actually add default image solution

alexdryden committed:
Fix previous commit

Remove the nav image fix as that is not what was wrong. Add the file.phtml
with a clearfix div after the image.

alexdryden commented:
@Bosterm doh! I made the fix to the Omeka copy of the file. Fixed by adding `file.phtml` with the fix. Also updated the cPanel instance so [that page](https://pressbookstest.web.illinois.edu/omeka-s/s/test-site/page/test-2) should look right now

Bosterm commented:
Okay great now it works!
